cmd/installations/inspect.go

```diff
@@ -71,10 +80,33 @@ func (o *statusOptions) run(ctx context.Context, cmd *cobra.Command, log logr.Lo
 		o.namespace = namespace
 	}
 
+	if o.allNamespaces {
+		if len(o.installationName) != 0 {
+			return fmt.Errorf("the --all-namespaces option cannot be used when an installation name is provided")
+		}
+		o.namespace = "*"
+	}
 	if o.namespace == "" {
 		return fmt.Errorf("namespace was not defined. Use --namespace to specify a namespace")
 	}
 
+	switch o.omode {
+	case OUTPUT_YAML:
+		o.oyaml = true
+	case OUTPUT_JSON:
+		o.ojson = true
+	case OUTPUT_WIDE:
+		o.owide = true
+	case "":
```

Comment:
is this a case that can occur? if yes, we should handle it.

Reply:
It is "handled" :-D
This case occurs if the `-o` flag is not set. Then `o.omode` will be empty.
The empty case exists only so that the switch statement doesn't go into the `default` case - which throws an error - if no output mode is specified.
If you don't like the notation, I can instead wrap the `switch` statement into an `if` which checks whether the `-o` flag is set?

Reply:
i think its fine. maybe a oneline comment to explain this, since it seems unintuitive to me